assets/bingo/lang/de_de.json

```
@@ -1,17 +1,364 @@
+
+	"bingo.preferences.custom_hud.adjust_col0.title": "Komponente oberhalb Schnellzugriffsleiste positionieren, Links",
+	"bingo.preferences.custom_hud.adjust_col1.title": "Komponente oberhalb Schnellzugriffsleiste positionieren, Rechts",
+	"bingo.preferences.custom_hud.adjust_col.description": "Hier kannst Du Positionen von Komponenten tauschen, sie in eine andere Spalte verschieben oder spezifische Einstellungen machen.",
+	"bingo.preferences.custom_hud.adjust_col.disabled": "Es sind momentan keine Komponenten in diesem Anzeige Platz.",
+	"bingo.preferences.custom_hud.adjust_col.move_left.title": "◀",
+	"bingo.preferences.custom_hud.adjust_col.move_right.title": "▶",
+	"bingo.preferences.custom_hud.adjust_col.move_left_right.description": "Verschiebt Komponenten in den anderen Anzeige Platz.",
+	"bingo.preferences.custom_hud.adjust_col.move_up.title": "▲",
+	"bingo.preferences.custom_hud.adjust_col.move_up.description": "Tauscht die Position dieser Komponente mit der Komponente oberhalb.",
+	"bingo.preferences.custom_hud.adjust_col.move_down.title": "▼",
+	"bingo.preferences.custom_hud.adjust_col.move_down.description": "Tauscht die Position dieser Komponente mit der Komponente unterhalb.",
+	"bingo.preferences.custom_hud.adjust_col.remove.title": "Entfernen",
+	"bingo.preferences.custom_hud.adjust_col.remove.description": "Entfernt die Komponente aus dem HUD.",
+	"bingo.preferences.custom_hud.adjust_col.add_spacer": "Freiraum hinzufügn.",
+
+	"bingo.preferences.custom_hud.adjust_card.title": "Positionierung Bingo-Karte",
+	"bingo.preferences.custom_hud.adjust_card.description": "Hier kannst Du Einstellungen der Bingo-Karte ändern.",
+	"bingo.preferences.custom_hud.adjust_card.left_of_hotbar.title": "Verschieben auf linke Seite der Schnellzugriffsleiste.",
+	"bingo.preferences.custom_hud.adjust_card.left_of_hotbar.description": "Verschiebt die Bingo-Karte auf die linke Seite der Schnellzugriffsleiste.",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_no_attack.title": "Verschieben auf rechte Seite der Schnellzugriffsleiste.",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_no_attack.description": "Verschiebt die Bingo-Karte auf die rechte Seite der Schnellzugriffsleiste ohne Raum für die Angriffsanzeige.\nFalls “Optionen → Grafikeinstellungen → Angriffsanzeige” auf “Schnellzugr.” steht, wähle bitte die nächste Option.",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_with_attack.title": "Verschieben auf rechte Seite der Schnellzugriffsleiste (Angriffsanzeige)",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_with_attack.description": "Verschiebt die Bingo-Karte auf die rechte Seite der Schnellzugriffsleiste mit Freiraum für die Angriffsanzeige.\nWähle diese Option nur falls “Optionen → Grafikeinstellungen → Angriffsanzeige” auf “Schnellzugr.” steht",
```

```suggestion
	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_no_attack.description": "Verschiebt die Bingo-Karte auf die rechte Seite der Schnellzugriffsleiste ohne Raum für die Angriffsanzeige.\nFalls „Optionen → Grafikeinstellungen → Angriffsanzeige“ auf „Schnellzugr.“ steht, wähle bitte die nächste Option.",
	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_with_attack.title": "Verschieben auf rechte Seite der Schnellzugriffsleiste (Angriffsanzeige)",
	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_with_attack.description": "Verschiebt die Bingo-Karte auf die rechte Seite der Schnellzugriffsleiste mit Freiraum für die Angriffsanzeige.\nWähle diese Option nur falls „Optionen → Grafikeinstellungen → Angriffsanzeige“ auf „Schnellzugr.“ steht",
```
German quotation marks